src/main/java/uk/gov/hmcts/reform/ccd/document/am/model/CaseDocumentsMetadata.java

```diff
@@ -28,6 +28,6 @@
     private String jurisdictionId;
 
     @JsonProperty("caseDocuments")
-    private List<Document> documents;
+    private List<CaseDocument> documents;
```

Comment:
Can we rename this field like "caseDocuments" instead of "documents" to maintain consistency as we have another entity called "Documents" for other purpose?

Reply:
yes this can be done.

Reply:
I have renamed this entity to DocumentToken instead of caseDocument.